lib/flutter_circle_color_picker.dart

```diff
@@ -79,6 +81,16 @@ class CircleColorPicker extends StatefulWidget {
   /// Default is Text widget that shows rgb strings;
   final ColorCodeBuilder? colorCodeBuilder;
 
+  /// Determines whether to draw the [colorCodeBuilder]
+  ///
+  /// Default value is false
+  final bool disableColorCode;
```

Comment:
You can disable color code widget by passing `SizedBox.shrink()` to `colorCodeBuilder`, so I think we don't need this property.

Reply:
@itome That's a good call. Is the `disableLightnessSlider` still ok?